judgments/tests/test_atom_feed.py

```diff
@@ -91,17 +91,29 @@ def test_search_query_in_URL(self, mock_api_client, mock_search_judgments_and_pa
         )
 
     @patch("judgments.feeds.search_judgments_and_parse_response")
-    def test_bad_page_not_500(self, mock_search_judgments_and_parse_response):
+    def test_bad_page_not_500(self, mock_search):
         # "?page=" doesn't 500
-        mock_search_judgments_and_parse_response.return_value = FakeSearchResponse()
+        mock_search.return_value = FakeSearchResponse()
         response = self.client.get("/atom.xml?page=")
         assert response.status_code != 500
 
     @patch("judgments.feeds.search_judgments_and_parse_response")
-    def test_feed_with_empty_date(self, mock_search_judgments_and_parse_response):
-        mock_search_judgments_and_parse_response.return_value = FakeSearchResponse()
+    def test_feed_with_empty_date(self, mock_search):
+        mock_search.return_value = FakeSearchResponse()
 
         response = self.client.get("/atom.xml")
         decoded_response = response.content.decode("utf-8")
         self.assertEqual(response.status_code, 200)
         self.assertIn("A SearchResult name!", decoded_response)
+
+    def test_redirect_full(self):
+        response = self.client.get("/ewhc/ch/2024/atom.xml")
```

Comment:
Can we include a test case which covers passing in page and ordering parameters as well?

Reply:
Done.